pkg/rhcos/builds.go

```diff
@@ -36,42 +34,65 @@ type metadata struct {
 	OSTreeVersion string `json:"ostree-version"`
 }
 
-func fetchLatestMetadata(ctx context.Context, channel string) (metadata, error) {
-	build := buildName
-	var err error
-	if build == "" {
-		build, err = fetchLatestBuild(ctx, channel)
+// rhcosBuildID is a version number e.g. 410.8.20180312.0
+var rhcosBuildID string
+
+// rhcosBuildChannel is a name for a stream of builds
+var rhcosBuildChannel string
+
+func fetchRHCOSBuild(ctx context.Context) (metadata, error) {
```

Comment:
this needs to be rewired as the overrides were dropped.

Reply:
I kept most of the code to handle multiple just dropped the override environment variables.  What do you want exactly?  Delete everything except support for reading the pinned JSON?

Reply:
> Delete everything except support for reading the pinned JSON?

We still need a way to override for CI, but that override does not need any build fetching, so this signature looks fine to me.